presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -819,3 +819,26 @@ row. The returned value is ``NULL`` if the subquery produces no rows::
     WHERE regionkey = (SELECT max(regionkey) FROM region)
 
 .. note:: Currently only single column can be returned from the scalar subquery.
+
+LATERAL
```

Comment:
I think we should move this above to the joins section, probably after CROSS JOIN. While a lateral join contains a subquery, it's not a subquery expression, so it fits better with joins.